roles/openshift_master_facts/test/openshift_master_facts_default_predicates_tests.py

```diff
@@ -153,9 +157,12 @@ def deployment_type_kwarg_fixture(request, facts):
     return facts, deployment_type, default_predicates
 
 
-def test_short_version_deployment_type_kwargs(predicates_lookup, short_version_deployment_type_kwargs_fixture, regions_enabled):
+def test_short_version_deployment_type_kwargs(
+        predicates_lookup, short_version_deployment_type_kwargs_fixture, regions_enabled):
     short_version, deployment_type, default_predicates = short_version_deployment_type_kwargs_fixture
-    assert_ok(predicates_lookup, default_predicates, regions_enabled=regions_enabled, short_version=short_version, deployment_type=deployment_type)
+    assert_ok(
+        predicates_lookup, default_predicates, regions_enabled=regions_enabled,
+        short_version=short_version, deployment_type=deployment_type)
```

Comment:
Nope. This was manual, just to reduce the number of columns to less than 120 (our limit in pylint).

The line length is currently ignored by the CI job because we do not run pylint on tests, #3511 is addressing that.